packages/lesswrong/server/vulcan-lib/mutators.ts

```diff
@@ -293,7 +293,8 @@ export const updateMutator = async <T extends DbObject>({
 
   // get a "preview" of the new document
   let document: T = { ...oldDocument, ...data };
-  document = pickBy(document, f => f !== null);
+  // FIXME: This replaces all null fields with undefined, which seems wrong. But maybe there's a reason for it?
```

Comment:
Hmm, this sure seems risky. I don't know how I feel about this.

Reply:
For now, leaving this out with an explicit cast and a FIXME.